bin/fun.js

```diff
@@ -23,7 +23,9 @@ program
   .command('build', 'build the dependencies')
   .command('local', 'run your serverless application locally')
   .command('validate', 'validate a fun template')
-  .command('deploy', 'deploy a fun application');
+  .command('deploy', 'deploy a fun application')
+  .command('init', 'initializes a new fun project');
```

Comment:
```suggestion
  .command('init', 'initialize a new fun project');
```
All subcommands in this overview view are described in the second person.

Reply:
ok